cpa/tester/policies/helpers/orbs/allowlist/policy.rego

```diff
@@ -0,0 +1,14 @@
+package org
+
+import data.circleci.config
+import future.keywords
+
+policy_name["orbs_allowlist"]
+
+enable_rule["orbs_allowlist"] {
```

Comment:
have you not heard of `enable_hard`? It should be released in the docs as of today :) and has been live for a week or two